packages/nest/src/exclusive-field.validator.test.ts

```diff
@@ -0,0 +1,39 @@
+import { describe, expect, it } from '@jest/globals';
+import { InputType } from '@nestjs/graphql';
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import { ExclusiveFieldExists } from './exclusive-field.validator';
+
+describe('ExclusiveFieldExists', () => {
+  it('should return if only one of the fields exist', () => {
+    const ability = CreateCharacter({ ability: { strength: 10 } });
+
+    expect(ability.strength).toBe(10);
+  });
+
+  it('should return specific error message if zero fields exist', () => {
+    const ability = CreateCharacter({ ability: {} });
+
```

Comment:
@CarsonF - Perhaps this entire approach of testing is wrong...could you throw a bone?  It actually enters in here, except it never returns the error message.  Do I actually have to create a Resolver for this to work?

Reply:
Check here
https://github.com/typestack/class-validator/blob/develop/test/functional/custom-decorators.spec.ts